<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>749</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QFrame" name="first_mediaplayer">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>0</y>
      <width>461</width>
      <height>301</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QSlider" name="time_slider">
     <property name="geometry">
      <rect>
       <x>58</x>
       <y>240</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="current_time_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>248</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0:00</string>
     </property>
    </widget>
    <widget class="QLabel" name="time_left_label">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>248</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0:00</string>
     </property>
    </widget>
    <widget class="QVideoWidget" name="video_player" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>19</y>
       <width>421</width>
       <height>211</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QSlider" name="volume_control">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>262</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pauseButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>260</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>resources/buttons/media_buttons/pause1.png</normaloff>resources/buttons/media_buttons/pause1.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>31</width>
       <height>31</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="playButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>260</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>resources/buttons/media_buttons/play1.png</normaloff>resources/buttons/media_buttons/play1.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>31</width>
       <height>31</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="nextButton">
     <property name="geometry">
      <rect>
       <x>229</x>
       <y>260</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>resources/buttons/media_buttons/next_vid1.png</normaloff>resources/buttons/media_buttons/next_vid1.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>31</width>
       <height>31</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="stopButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>260</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>resources/buttons/media_buttons/stop1.png</normaloff>resources/buttons/media_buttons/stop1.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>31</width>
       <height>31</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="muteButton">
     <property name="geometry">
      <rect>
       <x>278</x>
       <y>262</y>
       <width>23</width>
       <height>23</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>resources/buttons/media_buttons/mute1.png</normaloff>resources/buttons/media_buttons/mute1.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>31</width>
       <height>31</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="volumeButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>261</y>
       <width>23</width>
       <height>23</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>resources/buttons/media_buttons/volume1.png</normaloff>resources/buttons/media_buttons/volume1.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>31</width>
       <height>31</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="mini_viewer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>991</width>
      <height>411</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGraphicsView" name="timeline_1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>891</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QGraphicsView" name="mini_view_1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>891</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QGraphicsView" name="timeline_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>210</y>
       <width>891</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QGraphicsView" name="mini_view_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>320</y>
       <width>891</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QPushButton" name="push_extract_1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Extract Videos</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="timeline_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>250</y>
       <width>891</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QGraphicsView" name="timeline_4">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>290</y>
       <width>891</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>511</width>
      <height>301</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <attribute name="title">
      <string>Original Videos</string>
     </attribute>
     <widget class="QListView" name="playlistView">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>511</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <attribute name="title">
      <string>Extracted Videos</string>
     </attribute>
     <widget class="QTreeView" name="treeView2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>511</width>
        <height>271</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Model</string>
     </attribute>
     <widget class="QTreeView" name="treeView2_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>511</width>
        <height>271</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>112</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>999</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>0</number>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionLicense"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="actionAdd_Files"/>
   </widget>
   <addaction name="menuFiles"/>
   <addaction name="menuView"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionLicense">
   <property name="text">
    <string>License</string>
   </property>
  </action>
  <action name="actionAdd_Files">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add Files</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>PyQt5.QtMultimediaWidgets</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
